### PR TITLE
World Air Quality with React & Redux 🚀 

### PR DESCRIPTION
**Pull request by hsztan**
I created a SPA with react to show 👀 the air quality 💨 of all countries and major cities 🌏. I use `useParams` hook to send parameters data 💾 to components to filter information the user wants. Air quality is displayed on each list, and the last page shows the city population 👨‍👩‍👦‍👦 and AQI index. 😎

> [video recording 🎥](https://www.loom.com/share/7ca8df4ad3374b43a8f6f30db1170900)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,3 +84,37 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
...
+
+- GitHub: [@hsztan](https://github.com/hsztan)
+- Twitter: [@hsztanski](https://twitter.com/hsztanski)
+- LinkedIn: [henry-nawrocki](https://linkedin.com/in/henry-nawrocki)
+
+👤 **Dangelo Arrivillaga**
+
+- GitHub: [@Dangelo-JAN](https://github.com/Dangelo-JAN)
+- LinkedIn: [Soluciones Empresriales Dangelo Arrivillaga](https://www.linkedin.com/in/soluciones-empresariales-dangelo-arrivillaga-2a144718a/)
+
```
Comment (Whoistolu) [inline README.md]: - [x] [REQUIRED] Let us ensure that we have just the details of the person that built the project here. This is a project built by just you but we have details for two developers in the README.

File: README.md [hunk 2]
```diff
@@ -68,3 +84,37 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
...
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (Whoistolu) [inline README.md]: - [x] [REQUIRED] Kindly ensure that you give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @hsztan 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (hsztan) [reply README.md]: Absolutely right. Thank you for your feedback 😎

File: README.md (hunk 2 above)
Comment (hsztan) [reply README.md]: Will certainly do 😎

File: README.md (hunk 1 above)
Comment (hsztan) [reply README.md]: I corrected my README.md

File: README.md (hunk 2 above)
Comment (hsztan) [reply README.md]: I added the acknowledgment to Nelson Sakwa 🤗

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @hsztan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### You did well to make the changes requested by the previous reviewer ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**